views/layout.hbs

```diff
@@ -9,8 +9,17 @@
     <link rel="stylesheet" href="/stylesheets/style.css" />
   </head>
 
-  <body>
+  <body class="wallpaper">
 
+    <nav>
+      <a href="/">Home</a>
+      <a href="/signup">Signup</a>
+      <a href="/login">Login</a>
+      
```

Comment:
When doing the lab, try adding `<a>` tags to all your routes for easier navigation!